benchmarks/passing/bril/core/fact.bril

```diff
@@ -1,27 +1,10 @@
-# ARGS: 9500
```

Comment:
why's the input smaller now?

Reply:
It was overflowing or something, but it didn't need to be that big.